gallery/src/data-visualization/ReactVis.scss

```diff
@@ -0,0 +1,33 @@
+@import "~react-vis/dist/style";
```

Comment:
I assume the styles in this import are required when using these charts? We need to document that.